hathor/manager.py

```diff
@@ -911,11 +908,13 @@ def tx_fully_validated(self, tx: BaseTransaction) -> None:
             self.wallet.on_new_tx(tx)
 
     def listen(self, description: str, use_ssl: Optional[bool] = None) -> None:
-        endpoint = self.connections.listen(description, use_ssl)
+        self.connections.listen(description, use_ssl)
+        # XXX: although endpoint._port has the port it is private/internal, description also has the port
```

Comment:
No, mostly because the way endpoint is typed, it may not even have a port depending on the concrete class (for example, it could be listening on a unix socket, which does not have a port).

What do you think about using `getattr(endpoint, `_port`, None)`?

Reply:
I see. The `listen(...)` method accepts many different descriptions. I feel it's safer to use `getattr(endpoint, '_port', None)`.

Reply:
Done.

Reply:
Thanks!